pkg/openapi/api_violations.report

```diff
@@ -10,7 +10,9 @@ API rule violation: names_match,github.com/gardener/gardener/pkg/apis/garden/v1b
 API rule violation: names_match,github.com/gardener/gardener/pkg/apis/garden/v1beta1,KubeControllerManagerConfig,HorizontalPodAutoscalerConfig
 API rule violation: names_match,github.com/gardener/gardener/pkg/apis/garden/v1beta1,KubeLego,Mail
 API rule violation: names_match,github.com/gardener/gardener/pkg/apis/garden/v1beta1,KubeletConfig,PodPIDsLimit
+API rule violation: names_match,github.com/gardener/gardener/pkg/apis/garden/v1beta1,KubernetesStatus,KubeProxy
```

Comment:
I'd stick to the API naming conventions if possible rather than adding it as an exception.